e2e/test/scenarios/permissions/sandboxing/sandboxing-via-ui.cy.spec.ts

```diff
@@ -47,9 +43,6 @@ describe(
       cy.intercept("/api/card/*/query").as("cardQuery");
 
       H.restore("postgres-12");
-      cy.signInAsAdmin();
-      H.setTokenFeatures("all");
-      preparePermissions();
```

Comment:
I think removing this signin is causing this 401 error

https://github.com/metabase/metabase/actions/runs/14000117404/job/39204922085?pr=54684#step:14:305

Not sure if you moved this into a helper, but as more stuff gets moved into helpers it's harder to figure out who is signed in when